tests/test_ui_domain_view.py

```diff
@@ -323,120 +299,15 @@ def view13(test_data):
     )
 
 
-@pytest.fixture()
-def view14(test_data, mock_ipwhois_get, mock_urlhaus_get):
-    """Same as view01() but with Urlhaus enrichment. Test URLhaus only."""
-    resolutions = Resolutions.model_validate(
-        json.loads(test_data("vt_resolutions_gist.json"))
-    )
-
-    geoasn_pool = json.loads(test_data("ipwhois_gist.json"))
-    geoasn_client = IpWhoisClient()
-    geoasn_client._get_ipwhois = MagicMock(
-        side_effect=lambda ip: mock_ipwhois_get(ip, geoasn_pool)
-    )
-    geoasn_enrich = geoasn_client.enrich_ips(*resolutions.ip_list(3))
-
-    urlhaus_pool = json.loads(test_data("urlhaus_gist.json"))
-    urlhaus_client = UrlHausClient()
-    urlhaus_client._get_host = MagicMock(
-        side_effect=lambda domain: mock_urlhaus_get(domain, urlhaus_pool)
-    )
-    urlhaus_enrich = urlhaus_client.enrich_domains("gist.github.com")
-
-    return DomainView(
-        console=Console(),
-        entity=Domain.model_validate(json.loads(test_data("vt_domain_gist.json"))),
-        resolutions=resolutions,
-        geoasn=geoasn_enrich,
-        whois=PTWhois.model_validate(json.loads(test_data("pt_whois_gist.json"))),
-        shodan=ShodanIpMap.model_validate({}),
-        greynoise=GreynoiseIpMap.model_validate({}),
-        abuseipdb=AbuseIpDbMap.model_validate({}),
-        urlhaus=urlhaus_enrich,
-    )
```

Comment:
We still need this fixture for URLhaus. As before, VT can be used for the whois.